deliverables/README.md

```diff
@@ -1,14 +1,23 @@
 ## Deliverables
 #### Required
 ##### Questions
-- What is a framework?
-- What's the difference between a programming language and a framework?
-- What’s the difference between a library and a framework?
-- What languages have frameworks?
+
+###### What is a framework?  
+-   A framework provides an environment made of reusable boilerplate code that allows streamlining development           processes as well as enforces best practices 
+
+
+###### What's the difference between a programming language and a framework?
+-   A framework is a collection of abstractions written in a specific language, and a language is what the framework     is built with
+
+###### What’s the difference between a library and a framework?  
+-   A library provides specific, low-level functionality. Example: imagemagick 
```

Comment:
A library is actually a reusable collection of functions like jQuery is for JavaScript